release/mixer.sh

```diff
@@ -232,7 +232,8 @@ fi
 
 log_line "Checking Downstream Repo:"
 if [[ -n "$(ls -A "${PKGS_DIR}")" ]];then
-    mixer_cmd repo set-url "${CONTENT_REPO}" "file://${REPO_DIR}/x86_64/os" > /dev/null
+    mixer_cmd config set Mixer.LOCAL_RPM_DIR "${REPO_DIR}/x86_64/os/Packages"
```

Comment:
This should be PKGS_DIR as well, it was breaking when you actually have custom content. I fixed and merged this.